third_party/react-externs/externs.js

```diff
@@ -150,13 +150,13 @@ React.Component.prototype.context;
 
 /**
  * @type {Object}
- * @protected
+ * @package
```

Comment:
Not sure we should be modifying code in third_party. Do these generate errors? I thought we did some selective filtering of errors in third_party.

Reply:
Reverted.